preprocessing.py

```diff
@@ -210,7 +220,12 @@ def get_tpu_input_tensors(batch_size, tf_records, num_repeats=1,
         functools.partial(batch_parse_tf_example, batch_size))
 
     if random_rotation:
-        raise ValueError("random_rotation not yet supported on TPU")
+        # Unbatch the dataset so we can rotate it
```

Comment:
I don't mind if a minibatch uses exactly the same rotation - all that I care about is that the 4 positions pulled from 1 game get assigned different rotations. And since the 4 positions are highly unlikely to be in the same minibatch, they're likely to be assigned randomly different rotations. Unbatching might have undesirable performance side effects.

Reply:
Andrew ran this last night and I there's no difference in performance.